pkg/common/security/usergroup_test.go

```diff
@@ -50,9 +51,24 @@ func (c *UserGroupCache) getUGmap() map[string]*UserGroup {
 	return c.ugs
 }
 
+// UserGroupResolver Config for the test
+var testResolver = configs.UserGroupResolver{
+	Type: "test",
+}
+
+// UserGroupResolver Config for the os resolver
+var osResolver = configs.UserGroupResolver{
+	Type: "os",
+}
+
+// UserGroupResolver Config for the unknown resolver
+var unknownResolver = configs.UserGroupResolver{
+	Type: "unknown",
+}
+
```

Comment:
Please see earlier comment. We need to run these tests even for this new resolver.